test/test_client.py

```diff
@@ -106,7 +107,7 @@ def setUpClass(cls):
 
     @classmethod
     def tearDownClass(cls):
-        cls.client.close()
```

Comment:
I think this close should still be here, why did you choose to remove it?

Reply:
I think you are right--I reverted it.